tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.TestBot/README.md

```diff
@@ -0,0 +1,106 @@
+﻿# EchoBot using Facebook Adapter
+
+Bot Framework v4 echo bot using facebook Adapter sample.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to create a simple echo bot that connects with Facebook to respond to messages.
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 2.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+## To try this sample
+
+1 - Clone the repository
+
+   ```bash
+    git clone https://github.com/Microsoft/botbuilder-dotnet.git
+   ```
+
+ 2 - In a terminal, navigate to `test/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.TestBot`
+
+ 3 - Connect the bot with Facebook by following the instructions below.
+
+4 - Run the bot from a terminal or from Visual Studio, choose option A or B.
+
+  A) From a terminal
+
+  ```bash
+  # run the bot
+  dotnet run
+  ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `test/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.TestBot` folder
+  - Select `Microsoft.Bot.Builder.Adapters.Facebook.TestBot.csproj` file
+  - Press <kbd>F5</kbd> to run the project
+
+
+### Connect the bot with Facebook
+
+1 - Create a Facebook Account for Developers (https://developers.facebook.com/).
+
+2 - Create a new App. Give a name to the app and click Create App ID button.
+
+3 - In the Dashboard go to Add a Product and select Messenger by clicking on Set Up button
+
+    A) In the Access Tokens section, select a Facebook Page or create a new one. This is the page where the bot will be tested later.
+
+    B) After selecting the page, the permissions should be edited, select the "Tutorial Page" option and continue.
```

Comment:
The "Tutorial Page" was the fantasy name of the page we used to create the tutorial.